be/CMakeLists.txt

```diff
@@ -403,9 +403,6 @@ if (WITH_GCOV)
     set(CXX_GCC_FLAGS "${CXX_GCC_FLAGS} -fprofile-arcs -ftest-coverage")
 endif()
 
-# to compresss debug section. https://gcc.gnu.org/onlinedocs/gcc/Debugging-Options.html
-set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -gz=zlib")
```

Comment:
Removing this line have the side-effect？

Reply:
The compressed debug info is not recognized